tests/VObject/Component/VAvailabilityTest.php

```diff
@@ -236,7 +236,7 @@ public function testAvailableSubComponent()
 VCAL;
         $document = Reader::read($vcal);
 
-        $this->assertInstanceOf(__NAMESPACE__, $document->VAVAILABILITY->AVAILABLE);
+        $this->assertInstanceOf(Available::class, $document->VAVAILABILITY->AVAILABLE);
```

Comment:
This looks like a error

Reply:
I did this on purpose, although it wasn't necessarily wrong before.

Before, this asserted that the object was an instance of `Component` as that is the namespace which the test is in. However, there is also a class Component which happens to share it's class path with the Component namespace. If I'd want to reference the Component class instead of the namespace, I'd need to add a `use` statement. Not too much trouble, but the test would most probably like to assert that the object is an instance of `Available`, which is what you'd expect. Therefore, I chose to do that instead.